models/place-mediaObject.json

```diff
@@ -26,10 +26,10 @@
         "pattern": "^http[s]?:\\/\\/",
         "example": "https://io.uitdatabank.be/place/85b04295-479c-40f5-b3dd-469dfb4387b3",
         "description": "A unique URI that acts as globally unique ID for the media object. Based on JSON-LD node identifiers. https://www.w3.org/TR/json-ld/#node-identifiers",
-        "readOnly": true,
         "examples": [
           "https://io.uitdatabank.be/place/85b04295-479c-40f5-b3dd-469dfb4387b3"
-        ]
+        ],
+        "readOnly": true
```

Comment:
Nit: I think `@type`, `contentUrl` and `thumbnailUrl` can also be made `readOnly`.

Reply:
\+ `videos[].embedUrl`

Reply:
\+ `availableTo`
\+ `created`
\+ `creator`
\+ `modified`
\+ `sameAs`

Reply:
I would also suggest to move these down in the list of properties, so they only appear at the end.

Reply:
I marked those properties as readonly and moved them lower. But I kept the `availableTo` property underneath the `availableFrom` one.